api-service/src/controllers/CreateQueryTemplate/QueryTemplateValidator.ts

```diff
@@ -0,0 +1,58 @@
+import * as _ from "lodash";
+
+const template_required_variables = process.env.template_required_vars ? process.env.template_required_vars.split(",") : ["DATASET", "STARTDATE", "ENDDATE",];
```

Comment:
This file looks like copied from old code but need to follow new standards like logging and refactoring code to reduce the number of lines

Reply:
resolved